### PR TITLE
Feature/ev 239 partner organizers

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

https://jira.itkdev.dk/browse/EV-239 (to support https://jira.itkdev.dk/browse/DOKK-1170)

#### Description

Added "Partner Organizers" to Event

#### Screenshot of the result

API

Screenshot 2023-03-23 at 11 22 13

Admin

Screenshot 2023-03-23 at 11 22 26

#### Checklist

- [x] My code is covered by test cases.
- [x] My code passes our test (all our tests).
- [x] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: app/Resources/translations/messages.da.yml [hunk 1]
```diff
@@ -62,6 +63,7 @@ app:
                 start_day: Dag hvor begivenheden starter
                 end_day: Dag hvor begivenheden slutter
             organizer: Vælg arrangør
+            partnerOrganizers: Vælg yderligere arrangører der er partnere i arrangemenet
```
Comment (rimi-itk) [inline app/Resources/translations/messages.da.yml]: Typo: “arrangemenet” should be “arrangementet”.

File: app/config/easyadmin.yml [hunk 2]
```diff
@@ -271,6 +277,8 @@ easy_admin:
                       label: app.event.name
                     - property: organizer
                       label: app.event.organizer
+                    - property: partnerOrganizers
+                      label: app.event.partnerOrganizers
```
Comment (rimi-itk) [inline app/config/easyadmin.yml:281]: Is `help: app.event.help.partnerOrganizers` missing here (by mistake)?

File: features/events.organizer.feature [hunk 3]
```diff
@@ -33,6 +45,11 @@ Feature: Events
     And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
     And the JSON node "organizer.name" should be equal to "The organizer"
     And the JSON node "organizer.@id" should be equal to "/api/organizers/1"
+    And the JSON node "partnerOrganizers" should have 2 element
```
Comment (rimi-itk) [inline features/events.organizer.feature]: Consider pluralising “element” here.

File: features/events.organizer.feature [hunk 4]
```diff
@@ -115,7 +132,7 @@ Feature: Events
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
-    And the JSON node "hydra:member" should have 1 element
+    And the JSON node "hydra:member" should have 3 element
```
Comment (rimi-itk) [inline features/events.organizer.feature]: Consider pluralising “element” here.

File: src/AppBundle/Entity/Event.php [hunk 5]
```diff
@@ -398,6 +420,33 @@ public function getOrganizer()
         return $this->organizer;
     }
 
+    public function addPartnerOrganizers($partnerOrganizer)
```
Comment (rimi-itk) [inline src/AppBundle/Entity/Event.php]: This function should be called `addPartnerOrganizer` (singular “Organizer”), but some reflection magic somewhere may require it to be called `addPartnerOrganizers`.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: app/config/easyadmin.yml (hunk 2 above)
Comment (turegjorup) [reply app/config/easyadmin.yml:281]: No - this is the `show` configuration. None of the fields display the help text when "showing" the entity:

```
            show:
                fields:
                    - property: name
                      label: app.event.name
                    - property: organizer
                      label: app.event.organizer
                    - property: partnerOrganizers
                      label: app.event.partnerOrganizers
                    - property: isPublished
                      label: app.event.isPublished
                    - property: hasFullAccess
```

File: src/AppBundle/Entity/Event.php (hunk 5 above)
Comment (turegjorup) [reply src/AppBundle/Entity/Event.php]: Changed. Magic be dammed 🪄

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**